src/ontology/mondo-edit.obo

```diff
@@ -462876,6 +462878,3034 @@ name: neurological muscular channelopathy due to a genetic ryanodine receptor de
+intersection_of: MONDO:0700097 MONDO:0017987 ! cross-species analog syringomyelia
+
+[Term]
+id: MONDO:1010237
+name: polymicrogyria, non-human animal
+xref: OMIA:001100
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0000087 ! cross-species analog polymicrogyria
+
+[Term]
+id: MONDO:1010238
+name: leukodystrophy, non-human animal
+xref: OMIA:001130
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0019046 ! cross-species analog leukodystrophy
+
+[Term]
+id: MONDO:1010239
+name: peripheral neuropathy, non-human animal
+xref: OMIA:001150
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0005244 ! cross-species analog peripheral neuropathy
+
+[Term]
+id: MONDO:1010240
+name: Horner syndrome, non-human animal
+def: "Due to damage to the sympathetic nerves of the face - characterised by miosis (constriction of the pupil), ptosis (drooping of the upper eyelid), and anhidrosis (absence of sweating of the face)" [OMIA:001197]
+xref: OMIA:001197
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0001294 ! cross-species analog Horner syndrome
+
+[Term]
+id: MONDO:1010241
+name: giant axonal neuropathy, non-human animal
+xref: OMIA:001207
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0000128 ! cross-species analog giant axonal neuropathy
+
+[Term]
+id: MONDO:1010242
+name: Alexander disease, non-human animal
+xref: OMIA:001208
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0008752 ! cross-species analog Alexander disease
+
+[Term]
+id: MONDO:1010243
+name: cerebellar degeneration, non-human animal
+xref: OMIA:001221
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0022687 ! cross-species analog cerebellar degeneration
+
+[Term]
+id: MONDO:1010244
+name: L-2-hydroxyglutaricacidemia, non-human animal
+xref: OMIA:001371
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009370 ! cross-species analog L-2-hydroxyglutaric aciduria
+
+[Term]
+id: MONDO:1010245
+name: hyperekplexia, non-human animal
+synonym: "Startle disease" EXACT [OMIA:001594]
+xref: OMIA:001594
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0013827 ! cross-species analog hyperekplexia 3
```

Comment:
what is the source for cross-species analog "hyperekplexia 3"? is it based on orthologues?